M2/Macaulay2/m2/system.m2

```diff
@@ -112,6 +112,10 @@ setUpApplicationDirectory = () -> (
      f := (n,c) -> (n = dir|n; if not fileExists n then n << c << close);
      f("init.m2", sampleInitFile);
      f("README", readmeFile);
+     f(historyFilename,
+	 concatenate(
+	     "-- This is the beginning of your Macaulay2 log stored at ",
+	     historyFile, dir, historyFilename, newline));
```

Comment:
What is `historyFile`?  Should this line just be `dir, historyFilename, newline));`?

Reply:
Opps typo.